File: week-02/day-2/factorio.py
# - Create a function called `factorio`
#   that returns it's input's factorial

def factorio(x):
    if x == 0:
        return 1
    else:
        return x * factorio(x-1)

print(factorio(4))
print(factorio(5))
print(factorio(1))

File: week-07/day-02/factorio.js
'use strict';
// - Create a function called `factorio`
//   that returns it's input's factorial

function factorio(numberToFactor) {
    if (numberToFactor === 0 || numberToFactor === 1)
      return 1;
    for (var i = numberToFactor - 1; i >= 1; i--) {
      numberToFactor *= i;
    }
    return numberToFactor;
    }

console.log(factorio(5));